lms/djangoapps/django_comment_client/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls.defaults import url, patterns, include
 
-urlpatterns = patterns('',  # nopep8
-    url(r'forum/?', include('django_comment_client.forum.urls')),
+urlpatterns = patterns(
+    '',
+    url(r'forum/', include('django_comment_client.forum.urls')),
```

Comment:
OK, thanks. We might need to run this change by someone in the forums team. I'm not sure of the significance that the regex change might cause.